naughts_crosses.py

```diff
@@ -30,6 +30,30 @@ def prepareBoard():
 
     return board
 
+def drawBoard():
+    global board
+    global COLUMNS
+    global ROWS
+
+    print('| 1 | 2 | 3 | 4 | 5 | 6 |')
+    print('--------------------------------')
+
+    for row in range(ROWS):
+        rowHeight = ROWS - row
+        currentRow = '|'
+        for col in range(1, COLUMNS + 1):
+            if len(board[col]) < rowHeight:
+                currentRow += '   |'
+            elif board[col][rowHeight - 1] == 'O':
```

Comment:
I get what you're saying, but I think there's still potential to shoehorn in the use of player counters to avoid having hard-coded symbols. What if wanted it to become "exclamations and questions" rather than naughts/crosses? Or even load the symbols in from a file?